client/src/components/Navbar/index.jsx

```diff
@@ -0,0 +1,90 @@
+import {
+  Layout, Menu, Avatar, Dropdown,
+} from 'antd';
+import {
+  LogoutOutlined, UserOutlined,
+} from '@ant-design/icons';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import './style.css';
+import { Link } from 'react-router-dom';
+import logo from '../../assets/images/logo.png';
+import { LoginButton, SignupButton } from '../Forms';
+
+const { Header } = Layout;
+const { Item } = Menu;
+const Navbar = () => {
+  const { user } = useSelector((state) => state.auth);
```

Comment:
Destructure the id and image from the user and you can check the if there is a user from isAuthenticated,